public/client.js

```diff
@@ -132,30 +132,40 @@ socket.on('Display Message', function(data) {
         }
     }
     var dateTime = new Date();
+    var year = dateTime.getFullYear().toString(10);
+    var month = dateTime.getMonth().toString(10);
+    var day = dateTime.getDay().toString(10);
     var hours = dateTime.getHours().toString(10);
     var mins = dateTime.getMinutes().toString(10);
+    var secs = dateTime.getSeconds().toString(10);
     if (hours.length == 1) {
         hours = '0' + hours;
     }
     if (mins.length == 1) {
         mins = '0' + mins;
     }
+    if (secs.length == 1) {
+        secs = '0' + secs;
+    }
     // Format Message
     const div = document.createElement('div');
-    const username = document.createElement('small');
-    const timestamp = document.createElement('small');
-    const br = document.createElement('br');
-    div.classList.add("bubble", class_name);
-    username.classList.add("info");
-    timestamp.classList.add("info");
+    const username = document.createElement('div');
+    const timestamp = document.createElement('div');
+    const userMsg = document.createElement('p');
+    //const br = document.createElement('br');
+    div.classList.add("card", "bg-light", "mb-3", class_name);
```

Comment:
Here the `class_name` is not required because you are not using `float-left` or `float-right`. And also 'bg-light is not working because that is already the color of background.

Reply:
See comment above about the if-else statement. I also switched out .bubble in the style.css file for .card so that the floating and styling works again.